docker-compose.yml

```diff
@@ -19,6 +19,7 @@ services:
       - ./webapp:/usr/src/app/webapp
       - ./server:/usr/src/app/server
       - ./package.json:/usr/src/app/package.json
+      # - ./scripts:/usr/src/app/scripts
```

Comment:
@tynes this is something I use pretty frequently with debugging, but if we wanted to clear this file up, we could get rid of all but maybe one of these for reference.

Reply:
Maybe we have a `docker-compose-dev.yaml` and a `docker-compose.yaml`?
Or we could ask the person who forked and has their own docker setup for feedback